.github/workflows/integration-build-product.yml

```diff
@@ -1531,7 +1531,7 @@ jobs:
           ssh ${{ steps.infra_outputs.outputs.ssh_connect_string }} -i ~/.ssh/ci_priv_key.pem -o StrictHostKeyChecking=no '
               sudo systemctl restart docker
               sleep 3
-              docker run --gpus=all --rm nvcr.io/nvidia/cuda:11.6.2-base-ubuntu20.04 nvidia-smi 2>&1
+              docker run --gpus=all --rm nvcr.io/nvidia/cuda:12.4.1-base-ubuntu22.04 nvidia-smi 2>&1
```

Comment:
this step is still on 20.04 on a self hosted. not github hosted. dunno if this is going to work

Reply:
not sure, I checked and all tf-env are mentioning jammy/22.04 templates

Reply:
removed workflwo since it is not used